frontend/src/lib/components/PropertyFilters/components/TaxonomicPropertyFilter.scss

```diff
@@ -130,8 +140,15 @@
         display: flex;
         justify-content: space-between;
         overflow: hidden;
+        color: black;
+
+        .LemonIcon {
+            color: var(--muted-alt);
```

Comment:
I believe we're dropping instances of muted-alt. Icons should use the secondary text variable (I'm not sure what the variable name is but it maps to the hex color code #5F5F5F.

Reply:
Not sure why we need to do this? Generally overriding styles of a standardised component should be avoided at all costs. It is virtually impossible to maintain overtime as we no longer can be sure that component rendered as designed.

If the button is missing a color combination then ideally we should consult the Design System, update the component with the new supported option. If it is a truly special case then we should consider not using the built in icon rendering but adding your own icon as a child of the button and styling it there.

Appreciate this is a little different to how we did things before - just trying to make sure we move more and more to a sustainable frontend codebase 😉

Reply:
Happy to make this more code compliant, I've been referring to the design system while swapping out the filters but obviously I don't have the full scope of how we like to do things now since I haven't been working on the design system itself. 

If you have some time it would be great if you could take a look at how I can lemonize this entire thing better 🤗 I definitely need some help there since property filters are a bit of a doozy

Reply:
Otherwise it's preferable to get in what we have now so we can unblock https://github.com/PostHog/posthog/pull/11490 ... and then circle back to design systeming these property filters properly.

Reply:
@liyiy I'm gonna change a couple of things and then share the diff 👍

Reply:
I'm seeing other instances of us overriding `.LemonIcon` in our stylings, specifically in `CodeSnippet.scss`, `Insight.scss`, `AlertMessage.scss`, etc.. 

Can someone explain to me why those made sense to do but this does not? 🤗